.changeset/olive-geese-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@soundxyz/graphql-react-query': minor
+---
+
+New "customPages" option on useInfiniteQuery
```

Comment:
what does this help with? maybe add example in changeset so it acts as docs

Reply:
oh sorry didn't read this question, yeah I will add an explanation here, this is to be able to add completely arbitrary pages that are going to be added at the beginning of the orderedList returned value, this is used for optimistic messaging in vault